src/webview/html/__tests__/render-test.js

```diff
@@ -1,14 +1,11 @@
+/* @flow strict-local */
 import messageTypingAsHtml from '../messageTypingAsHtml';
+import * as eg from '../../../__tests__/lib/exampleData';
 
 describe('typing', () => {
   it('escapes values', () => {
-    expect(
-      messageTypingAsHtml('&<r', [
-        {
-          avatar_url: '&<av',
-          email: '&<e',
-        },
-      ]),
-    ).not.toContain('&<');
+    const user = eg.makeUser({ name: '&<av' });
```

Comment:
Does this still have the `&<` sequence in the avatar_url? One of the things this is testing is that that gets properly encoded.

Reply:
It currently does, meaning `eg.makeUser` has this for `avatar_url`:

```
`https://zulip.example.org/yo/avatar-${name}.png`,
```

But that's up to `exampleData` to choose. Might as well pin to a particular `avatar_url` and `email` here by clobbering those properties; I'll do that.